Gemfile.lock

```diff
@@ -0,0 +1,195 @@
+GIT
+  remote: https://github.com/puppetlabs/puppet.git
+  revision: 5d7a81d87fc72c7c9a48d6779ae6ec950b5fdfc9
+  ref: 6.x
+  specs:
+    puppet (6.26.0)
```

Comment:
Puppet 6.26.0 hasn't been released yet and that seems to be the cause of at least one of the failing tests

Reply:
the gemfile.lock was created from main with `bundle lock`